Library/Homebrew/extend/kernel.rb

```diff
@@ -359,7 +359,7 @@ def capture_stderr
   end
 
   def nostdout(&block)
-    if verbose?
+    if T.unsafe(self).verbose?
```

Comment:
This might benefit from the `if respond_to?(:verbose?)` above, but afaict it only has a [single](https://github.com/Homebrew/brew/blob/86aeac6/Library/Homebrew/formula_versions.rb#L50) call site anyway

Reply:
Maybe we can just remove it from `extend/kernel` and it to the one place it's used. Unless, of course, we think it will get more use in the future.

Reply:
Ok, I moved this into `FormulaVersions` (which also lets us remove the `T.unsafe` wrapper).